### ISSUE TITLE
IntellisenseParser does not detect base type

### ISSUE THREAD

**Issue by MindGirl** [madskristensen/TypeScriptDefinitionGenerator#5]
### Installed product versions
- Visual Studio: 2017 RC.4+26206.0 Enterprise
- This extension: 1.0.11

### Description
When generating the TypeScript definition for a derived type, the `IntellisenseParser` does not detect the base type, although the definition for the base type already has been generated. The newly created definition file contains the interface for the derived **and** the base type.
I noticed that the `IntellisenseParser` is looking for `ResponseModel.cs.d.ts` while the `GenerationService` creates the file `ResponseModel.d.ts`.

### Steps to recreate
1. In an web application create a type named `ResponseModel`
2. Generate TypeScript definition for `ResponseModel.cs`
3. The file `ResponseModel.d.ts` is generated
4. Create a second type named `DerivedResponseModel` and derive it from `ResponseModel`.
5. Generate TypeScript definition for `DerivedResponseModel.cs`
6. The file `DerivedResponseModel.d.ts` is generated

### Current behavior
The file `DerivedResponseModel.d.ts` has the following content:
```c#
declare module server {
	interface DerivedResponseModel extends ResponseModel {
	}
	interface ResponseModel {
	}
}
```

### Expected behavior
The file `DerivedResponseModel.d.ts` should have the following content:
```c#
declare module server {
	interface DerivedResponseModel extends ResponseModel {
	}
}
```

**Comment by raboud** [comment]
Please Try Build 1.1.24 to see if your issues are corrected.

**Comment by MindGirl** [comment]
Thanks, it works now.
